agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AzureFunctionsInitializer.java

```diff
@@ -77,6 +72,18 @@ public void lazyLoad() {
             System.getenv("APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL")));
   }
 
+  private static void disableBytecodeInstrumentation() {
+    Instrumentation instrumentation = InstrumentationHolder.getInstrumentation();
+    ClassFileTransformer transformer = ClassFileTransformerHolder.getClassFileTransformer();
+    if (instrumentation == null || transformer == null) {
+      return;
+    }
+    if (instrumentation.removeTransformer(transformer)) {
+      ClassFileTransformerHolder.setClassFileTransformer(null);
```

Comment:
ya, I initially thought `removeTransformer` was enough, but otel javaagent also uses ClassFileTransformerHolder.getClassFileTransformer() to transform lambda classes (which don't go through normal bytecode instrumentation process), so that's why also need to set that to `null`